src/hw10/Product.java

```diff
@@ -0,0 +1,29 @@
+package src.hw10;
+
+public class Product {
```

Comment:
You have 2 different types of products. Please use inheritance in order to achive it.